src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClientTest.java

```diff
@@ -133,15 +134,14 @@ public void should_throw_case_client_service_exception_when_unable_to_process_bo
                 .willReturn(aResponse().withBody(new byte[]{}).withStatus(400)));
 
         // when
-        CaseClientServiceException exception = catchThrowableOfType(
+        BadRequest exception = catchThrowableOfType(
             () -> client.updateCase(url(), existingCase(), exceptionRecordRequestData(), s2sToken),
-            CaseClientServiceException.class
+            BadRequest.class
         );
 
         // then
-        assertThat(exception.getStatus()).isEqualTo(BAD_REQUEST);
-        assertThat(exception.getResponse()).contains("No content to map due to end-of-input"); // because byte[]{}
-        assertThat(exception.getResponseRawBody()).isEmpty();
+        assertThat(exception.getStatusCode()).isEqualTo(BAD_REQUEST);
+        assertThat(exception.getResponseBodyAsString()).isEqualTo(""); // because byte[]{}
```

Comment:
Alternatively, we could setup the mock to return a byte array representing some string and assert that it's that string here. Empty string is a bit of a special value.